layouts/index.html

```diff
@@ -27,11 +27,11 @@ <h1 class="usa-display">{{ $translation.homepage.header }}</h1>
             </option>
             {{ if eq $.Site.Params.language "english" }}
               {{ range where .Data.Pages.ByTitle "Section" "register" }}
-                <option value="{{ replace .Title "." "" | lower | urlize }}">{{ title .Title }}</option>
+                <option value="{{ .Params.state_abbreviation | lower }}">{{ title .Title }}</option>
```

Comment:
This looks to only be handling the English language. What about the `else` block below? Also where is `.Params.state_abbreviation` coming from?